systems-management/nisysmgmt.yml

```diff
@@ -1554,6 +1554,7 @@ paths:
       operationId: CreateJob
       x-ni-privilege: Write
       x-ni-operation: createOrCancelJobs
+      x-ni-request-variables: [REMOTE_USER]
```

Comment:
Generally, when you request REMOTE_USER, you should also request USER_NAME. The REMOTE_USER will be an ID string that you would store for auditing purposes, while the USER_NAME is what you would show in a UI.